Tests/PackageSigningTests/SigningEntityTests.swift

```diff
@@ -63,7 +63,7 @@ final class SigningEntityTests: XCTestCase {
             throw XCTSkip("Skipping because 'REAL_SIGNING_IDENTITY_LABEL' env var is not set")
         }
         let identityStore = SigningIdentityStore(observabilityScope: ObservabilitySystem.NOOP)
-        let matches = await identityStore.find(by: label)
+        let matches = identityStore.find(by: label)
```

Comment:
I made these changes in https://github.com/apple/swift-package-manager/pull/6237 already. You don't need to cleanup files in `PackageSigningTests`.